cmd/kubeapps-apis/plugins/helm/packages/v1alpha1/server_test.go

```diff
@@ -845,9 +844,6 @@ func TestGetAvailablePackageDetail(t *testing.T) {
 
 	for _, tc := range testCases {
 		t.Run(tc.name, func(t *testing.T) {
-			currentExpectedPackage := tc.expectedPackage
-			currentExpectedPackage.PkgVersion = tc.expectedVersion
```

Comment:
The line above seemed to be added to *update* the expected package, rather than just having the expected package with the correct version (I assume because a pre-created expected detail package was used).